scripts/terminate.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Log file for tracking output
+LOGFILE="/var/log/bpf_monitor.log"
+
+# Step 1: Compile the binaries
+cd /linux-dev-env/bpf-progs
+make clean
+make -j"$(nproc)"
+
+# Step 2: Run the binaries in sequence
+echo "Starting /linux-dev-env/bpf-progs/loops.user in background..."
+./loops.user &  # Run first binary in background
+sleep 1         # Wait for 1 second
+echo "Starting /linux-dev-env/bpf-progs/trigger_loops.user in background..."
+./trigger_loops.user &  # Run second binary in background
```

Comment:
Are these binaries included? I don't see them.

Reply:
Right now, I run make before this script manually. I'm also making minor changes to the loader program, let me give it a shot first. Do you want me to include binaries or are we fine running make manually?

Reply:
Yes `make` should be done mainly, and binaries shouldn't be in a git repo. I should have phrased that better. Basically I don't see `loops.user.c` or `trigger_loops.user.c`, so I'm a bit confused on how these binaries would generate.